<?xml version="1.0" encoding="UTF-8"?>
<launch>
    
    <arg name="robotname" default="testbot" />
    <arg name="planname" default="stop" />
    <arg name="use_gui" default="false" />
    
    <group ns="$(arg robotname)">
        <param name="robot_name" value="$(arg robotname)" />
        <param name="tf_prefix" value="$(arg robotname)" />

        <node pkg="pnp_ros" type="pnp_node" name="pnp" args="" output="screen">
            <param name="plan_folder" value="$(find lasr_pnp_bridge)/plans/" />
            <param name="current_plan" value="$(arg planname)" />
            <param name="use_java_connection" value="$(arg use_gui)" />
        </node>

	<node pkg="lasr_pnp_bridge" type="lasr_pnp_bridge_node" name="LasrPNPBridge" args="" output="screen" />

	<node pkg="lasr_pnp_bridge" type="testServer.py" name="PNPExternalServer" args="" output="screen" />

    </group>
</launch>

